var/spack/repos/builtin/packages/tmux/package.py

```diff
@@ -57,6 +58,8 @@ class Tmux(AutotoolsPackage):
     depends_on("automake", when="@master")
     depends_on("autoconf", when="@master")
 
+    depends_on("yacc")
```

Comment:
If this is a new dependency that only applies for 3.4 onwards:
```suggestion
    depends_on("yacc", when="@3.4:")
```
If this is a dependency that only applies at build time, not link/run dependency:
```suggestion
    depends_on("yacc", type="build")
```